packages/shim/package.json

```diff
@@ -14,6 +14,9 @@
   "publishConfig": {
     "access": "public"
   },
+  "dependencies": {
+    "@sentry/types": "0.5.4"
```

Comment:
Going forward, we should probably also provide a rollup build without that dependency for our CDN, just like the `@sentry/browser` package. This would allow users to include a single javascript file to shim sentry instead of having to install multiple packages for use cases without sophisticated bundlers.